### PR TITLE
FossID: Add missing match type to the enum

### PR DESCRIPTION
**Pull request by nnobelis**
Signed-off-by: Nicolas Nobelis <nicolas.nobelis@bosch.io>

### CODE REVIEW

File: clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt [hunk 1]
```diff
@@ -21,6 +21,7 @@ package org.ossreviewtoolkit.clients.fossid.model.result
 
 enum class MatchType {
     NONE,
+    FAILED,
```
Comment (sschuberth) [inline clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt:24]: Just wondering, is there any ordering in place here? If not, how about adding a commit to sort these, e.g. simply alphabetically?

Comment (sschuberth) [reply clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt:23]: > FossID: Sort the match type enum

For completeness, this should say "FossID: Sort the match type enum *alphabetcially*".

I haven't checked, but are there other enums in the FOSSID code that should get sorted in the same run?

Comment (nnobelis) [reply clients/fossid-webapp/src/main/kotlin/model/result/MatchType.kt:23]: jep, I changed them all

File: clients/fossid-webapp/src/main/kotlin/model/identification/common/LicenseMatchType.kt [hunk 2]
```diff
@@ -20,5 +20,5 @@
 package org.ossreviewtoolkit.clients.fossid.model.identification.common
 
 enum class LicenseMatchType {
-    SNIPPET, FILE
+    FILE, SNIPPET
```
Comment (sschuberth) [inline clients/fossid-webapp/src/main/kotlin/model/identification/common/LicenseMatchType.kt:23]: One last nit: I like the fact that you did wrap the last enum, and I believe we should then wrap the lines in this enum, too, to consistently have them all wrapped.

**Review by sschuberth — APPROVED** [review]

**Pull request merged**